CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ for more details.
 
 * Removed support for passing a plain object to the `model` prop of Hoist Components (previously
   deprecated back in v58). Use the `modelConfig` prop instead.
+* Improved `selectEditor` to stop editing when the value changes, rather than when the user clicks
+  away from the cell, making it more intuitive for users. This only occurs in the absence of
+  `GridModel.fullRowEditing` and `Select.enableMulti`.
```

Comment:
I'd vote for dropping this last sentence - it's a huge changelog entry already, that that's a bit more detailed than a developer needs in this context